db/create_tables.py

```diff
@@ -0,0 +1,9 @@
+from src.models.meeting import Meeting
+
+def create_tables():
+    print("Creating DynamoDB tables if they don't exist...")
+    Meeting.create_table(wait=True)
```

Comment:
note: if https://github.com/nayaverdier/dyntastic/pull/32/ is merged and a new release made, this could be:
```suggestion
    Meeting.create_table(wait=True, billing_mode="PAY_PER_REQUEST")
```

Reply:
https://github.com/nayaverdier/dyntastic/pull/32 is merged. Will watch for a new release.

Reply:
Updated!